cloud/shared/bin/run

```diff
@@ -123,4 +123,8 @@ if [[ -n "${LOCK_TABLE_DIGEST_VALUE}" ]]; then
   args=("${args[@]}" "--lock-table-digest-value" "${LOCK_TABLE_DIGEST_VALUE}")
 fi
 
+if [[ -n "${ALLOW_POSTGRESQL_UPGRADE}" ]]; then
```

Comment:
Could [this](https://github.com/civiform/cloud-deploy-infra/blob/7179b40fcce1fbfae84b372d458cc7fb552bfed4/cloud/shared/bin/lib/config_loader.py#L97) be reused or is it different?

Reply:
Oh I think it may be the [load_infra_variables](https://github.com/civiform/cloud-deploy-infra/blob/7179b40fcce1fbfae84b372d458cc7fb552bfed4/cloud/shared/bin/lib/config_loader.py#L86) actually (which shouldn't require adding it to env var docs). I don't know if its ideal, but here is an example of where we use those variables in another spot https://github.com/civiform/cloud-deploy-infra/blob/d82d36d185fff3ba579ec899bea4edf6f69d5380/cloud/aws/templates/aws_oidc/app.tf#L149.

Reply:
Got it, thanks for the info! What if instead of a config variable we prompted them asking if they wanted to perform the postgres upgrade? Somewhat based on the slack discussion about people accidentally setting the variable and forgetting.